mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -57,10 +80,21 @@ CLASS net/minecraft/unmapped/C_zpccxxiz net/minecraft/util/math/MathHelper
 	METHOD m_ewxpwxnf floorMod (FF)F
 		ARG 0 dividend
 		ARG 1 divisor
+	METHOD m_fdftwqif lerp (FII)I
+		ARG 0 delta
+		ARG 1 start
+		ARG 2 end
 	METHOD m_fdvuyfhh lerpAngleDegrees (FFF)F
 		ARG 0 delta
 		ARG 1 start
 		ARG 2 end
+	METHOD m_fpdoqovb getInt (Ljava/lang/String;I)I
+		ARG 0 string
+		ARG 1 fallback
+	METHOD m_fwvsrufh rayIntersectsAABB (Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_hbcjzgoe;)Z
```

Comment:
because then you get bad names like AABB sneaking in ;-;

Reply:
that's the only name that bothered me out of the ones I copied